utils/collectd/files/lua-scripts/dsl.lua

```diff
@@ -157,7 +156,7 @@ local function read()
 			get_values(hostname, errors, far_errors, "far")
 			get_values(hostname, line_vars, down_line, "down")
 			get_values(hostname, line_vars, up_line, "up")
-			get_values(hostname, erb_vars, erb, "")
+			-- get_values(hostname, erb_vars, erb, "")
```

Comment:
Why did you comment that out? It's not in your description of the change?

Reply:
This was added in [master](commit https://github.com/openwrt/openwrt/commit/70729d345414cdf1463dec042811813ff9a94e7a ) and [openwrt-22.03](https://github.com/openwrt/openwrt/commit/70729d345414cdf1463dec042811813ff9a94e7a).
The branch openwrt-21.02 does not have this commit https://github.com/openwrt/openwrt/commits/openwrt-21.02/package/network/config/ltq-vdsl-app/src/src/dsl_cpe_ubus.c
So we should not comment this out because you made a pullrequest against the master branch.

Reply:
Thanks for the explanation. Updated the PR accordingly.